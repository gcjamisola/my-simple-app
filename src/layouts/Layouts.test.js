import React from 'react';
import { shallow } from 'enzyme';

import {PlainLayout} from './index';

describe('PlainLayout component', () => {
	it('should render the component', () => {
		const component = shallow(<PlainLayout />);
		expect(component).toMatchSnapshot();
	});
});

