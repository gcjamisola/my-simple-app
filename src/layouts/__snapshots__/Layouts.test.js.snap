// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlainLayout component should render the component 1`] = `
<styled.div>
  <Header />
  <Footer />
</styled.div>
`;
